server/routes/caseNotes/caseNotesController.ts

```diff
@@ -160,7 +162,24 @@ export default class CaseNotesController {
     }
     await this.interventionsService.addCaseNotes(accessToken, draft.data)
     await this.draftsService.deleteDraft(draft.id, { userId: res.locals.user.userId })
-    res.redirect(`/${loggedInUserType}/referrals/${referralId}/case-notes`)
+    res.redirect(`/${loggedInUserType}/referrals/${referralId}/add-case-note/confirmation`)
+  }
+
+  async newCaseNoteConfirmation(
```

Comment:
should we call this `addCaseNoteConfirmation` to sync up with the previous steps of the journey (which seem to be named `addCaseNoteX`?

Reply:
renamed